srcpkgs/python3-liblarch/template

```diff
@@ -0,0 +1,14 @@
+# Template file for 'python3-liblarch'
+pkgname=python3-liblarch
+version=3.0.1
+revision=1
+wrksrc=liblarch-${version}
+build_style=python3-module
+makedepends="python3-setuptools"
+depends="python3 python3-gobject gtk+3"
+short_desc="Library to handle data structures such as trees and acyclic graphs"
+maintainer="Bleznudd <bleznudd@pm.me>"
+license="LGPL-3.0-only"
+homepage="https://github.com/getting-things-gnome/liblarch"
+distfiles="${homepage}/archive/v${version}.tar.gz"
+checksum=3a866cc43873454b88ae5526207555aca1f0e16e3120577d6135ea13db98ca68
```

Comment:
On line 16, add
```sh
if [ "${CROSS_BUILD}" ]; then
	hostmakedepends="python3"
fi
```
as `python3-setuptools` is not on crossbuild.

Reply:
Many thanks, it seems to be working ok now